src/analysis/modals/ComputeModal/GcpComputeModal/GcpComputeModal.test.ts

```diff
@@ -451,7 +472,7 @@ describe('GcpComputeModal', () => {
       const user = userEvent.setup();
 
       const disk = getDisk();
-      const runtimeProps = { runtimeConfig: getJupyterRuntimeConfig({ diskId: disk.id, tool }) };
+      const runtimeProps = { runtimeConfig: getJupyterRuntimeConfig({ diskId: disk.id }), tool };
```

Comment:
fun fact, this little thing typescript caught means we were never actually testing the gcp  modal with multiple tools like we thought we were. We were including it in the wrong spot in the JSON and it was never used 😅